### PR TITLE
Added Aliases and Organized

### PR DESCRIPTION
**Pull request by Yoshizilla315**
Not really sure where the two metagross ones were supposed to go so I put them at the bottom of // pokemon. From what I understand, those are for ones where the first letter of the alias doesn't match the first letter of the Pokemon? If not, it's easily fixed.
EDIT: Also, these were tested to not be there on main so they shouldn't be on the list unless something hasn't been merged yet.

### CODE REVIEW

**Comment by xfix** [comment]
You may want to configure your text editor to automatically remove whitespace at end of lines when saving.

File: data/aliases.js [hunk 1]
```diff
@@ -587,6 +588,8 @@ exports.BattleAliases = {
 	"mechatyranitar2": "MT2",
 	"monica": "Giant",
 	"majin": "Spirit",
+	"gross": "Metagross",
```
Comment (xfix) [inline data/aliases.js]: There is an ordering to Pokemon here

Comment (Yoshizilla315) [inline data/aliases.js]: Yeah it's alphabetical I know, but I wasn't completely sure if putting them there was correct.

**Comment by Yoshizilla315** [comment]
Uh I didn't see that before my most recent fix xfix so let's pray that goes well. I was a bit more careful this time.

File: data/aliases.js [hunk 2]
```diff
@@ -61,6 +61,7 @@ exports.BattleAliases = {
 	"megamaw": "Mawile-Mega",
 	"megamedi": "Medicham-Mega",
 	"megagross": "Metagross-Mega",
+	"mgross": "Metagross-Mega",
```
Comment (xfix) [inline data/aliases.js]: Should be moved to be after `mgar`.

Comment (Yoshizilla315) [inline data/aliases.js]: This is true, will do if you don't do it before me lol

**Comment by Yoshizilla315** [comment]
Ok I'm new to this is closing it ok? I think I messed up now my last commit has to be checked again.

**Comment by DeathlyPlays** [comment]
@xfix bump

**Pull request merged**